src/components/cookie-banner/index.md.njk

```diff
@@ -121,6 +121,13 @@ When the user sets or changes their cookie preferences using the controls on the
 
 ## Research on this component
 
-We remove the visible focus indicator from when shifting focus to the cookie banner. We do this because the cookie banner is the first element on the page and the last thing the user interacted with prior to it being given focus.
+When the user accepts or rejects cookies, a replacement message will display. For example, "Your cookie preferences have been saved." The focus also shifts to this new message.
 
-So we’ve assumed that giving the cookie banner a visible focus indicator in these circumstances would be more likely to confuse users than to help them, especially as the element is not normally operable with a keyboard. However, more research is needed to find out if that assumption is right.
+However, a visible focus indicator does not appear around the replacement message. This is different from [what happens when a user does something in a notification banner](../notification-banner/#reacting-to-something-the-user-has-done).
+
+We decided to remove the visible focus indicator for a few reasons, as the replacement message:
+- is a `<div>`, so the user usually can’t interact with it using a keyboard
```

Comment:
This is also true for notification banner – I think it's still very relevant to the decision we made, but the leading couple of sentences are framing this as 'why this is different to the notification banner' so it doesn't quite read right to me.

I'd probably focus more on the fact that it's non-interactive / non-operable rather than the fact that it is a `<div>` – the fact that the user can't interact with it is more important than what type of HTML element we used.

Finally, the fact that the replacement message is only ever programatically focused – you can't 'tab to it' – also seems important here. Again, this is also true for notification banner as well.

Reply:
```suggestion
We decided to remove the visible focus indicator from the replacement message for a few reasons, as:
- the user cannot interact with it
- it's the first element, at the very top of the page
- it displays in place of the cookie message, which is the last thing the user interacted with
```